### ISSUE TITLE
Under access problem when using screen reader

### ISSUE THREAD

**Issue by zhou-zhoukang** [plateaukao/einkbro#365]
**What device and app version are you using**
 - Device: Pixel 4
 - OS: Android 13
 - EinkBro Version 11.6.1

**Describe the bug**
First thank you for the app. I appreciate its clean UI. I think there are accessibility issues for those with poor vision. Blind people is unable to learn how to use the button located in the bottom of the screen's navigation bar when using talkback(a screen reader), which may lead to confusion. More content description may helps a lot.

**To Reproduce**
1. Open the talkback.
2. Open the EinkBro.
3. Touch any button from the navigation bar. 
4. Talkback does not provide guidance on how to use the button.

**Screenshots**
![image](https://github.com/plateaukao/einkbro/assets/75771822/61a6e07c-273f-4e7f-9973-0b6ba6cde2fe)

**Comment by plateaukao** [comment]
@zhou-zhoukang 
I tried to add descriptions for toolbar icons in [v11.7.0](https://github.com/plateaukao/einkbro/releases/tag/v11.7.0)
